charts/templates/statefulset.yaml

```diff
@@ -25,10 +27,16 @@ spec:
       initContainers: 
       - name: configure-group
         image: "{{ .Values.initContainerImage.repository }}:{{ .Values.initContainerImage.tag }}"
+        volumeMounts:
```

Comment:
Check with @barkhachoithani to make sure you both use the same path for the secret volume.

Reply:
Yes I'll take the same as @barkhachoithani.